src/analytics/private/sqls/redshift/event-v2.sql

```diff
@@ -51,27 +51,30 @@ CREATE TABLE IF NOT EXISTS {{schema}}.event_v2 (
     app_install_source varchar(255),
     platform varchar(255),
     project_id varchar(255) not null,
-    screen_name varchar(255),
-    screen_id varchar(255),
-    screen_unique_id varchar(255),
-    previous_screen_name varchar(255),
-    previous_screen_id varchar(255),
-    previous_screen_unique_id varchar(255),
-    page_referrer varchar(65535),
-    page_referrer_title varchar(2048),
-    page_title varchar(2048),
-    page_url varchar(65535),
-    page_url_path varchar(65535),
-    page_url_query_parameters super,
-    hostname varchar(2048),
-    latest_referrer varchar(65535),
-    latest_referrer_host varchar(2048),
-    app_start_is_first_time bool,
-    previous_view_time_msec bigint,
-    previous_view_engagement_time_msec bigint,
-    entrances bool,
-    previous_app_version varchar(255),
-    previous_os_version varchar(255),
+    screen_view_screen_name  varchar(255),
+    screen_view_screen_id  varchar(255),
+    screen_view_screen_unique_id varchar(255),
+    screen_view_previous_screen_name varchar(255),
+    screen_view_previous_screen_id  varchar(255),
+    screen_view_previous_screen_unique_id varchar(255),
+    screen_view_previous_time_msec bigint, 
+    screen_view_engagement_time_msec bigint,
```

Comment:
for previous_time_msec and engagement_time_msec, why are we changing to have separate fields for page_view and screen_view?

Reply:
can we add more ua_fields? exspecially the fields that we can use to identify bots? mybe turn ua as an super field to store all the output from ua parser
<img width="1331" alt="image" src="https://github.com/awslabs/clickstream-analytics-on-aws/assets/95616569/adcbad3a-8184-45b9-b4ab-4747ee17b4f3">

Reply:
can we add source_ip in the process_info field? this could allow user to filter based on ip address (e.g., internal traffic)